File: Sunny/Stack/stack.py
class Stack: 
    __DefaultSize = 10
    stack = []
    top = -1
    def __init__(self, size = None):
        if(size == None):
            self.size = self.__DefaultSize
        else:
            self.size = size
    
    def push(self, element):
        self.top += 1
        if(len(self.stack) >= self.size):
            raise Exception("Stack is full")
        else:
            self.stack.append(element)
        
    def pop(self):
        if self.top != -1:
            self.top -= 1
        else:
            raise Exception("top is -1")
            
    def display(self):
        for i in range(0, self.top + 1): 
            print(self.stack[i])

# Custom Stack 

stack1 = Stack()

stack1.push(1)
stack1.push(2)
stack1.push(3)

stack1.display()

print("\n")

stack1.display()

print("\n")


# print(stack1.__DefaultSize)

File: Sunny/Stack/stack.c
#include<stdio.h>

void main(){
    printf("Enter the length of Stack : "); 
    int n; 
    scanf("%d \n", &n); 
    int stack[n], top = -1;

    for(int i = 0; i<n; i++){
        printf("Enter the element \n"); 
        scanf("%d", &stack[i]); 
    } 
    
    for(int i = 0; i<n; i++){
        printf("%d -> ", i); 
    }
}    
